pipelines/smartseq2_single_sample/README.md

```diff
@@ -110,7 +112,7 @@ Overall, the workflow is divided into two parts that are completed after an init
 
 ### Part 1: Quality Control Tasks
 #### 1.1 Align reads to the genome using HISAT2
-HISAT2 is a fast, cost-efficient alignment tool that can determine the presence of non-transcript sequences and true transcript sequences, taking into account the presence of single-nucleotide polymorphisms ([Kim et al.,2019](https://www.nature.com/articles/s41587-019-0201-4)). The Smart-seq2 Single Sample workflow uses the [HISAT2 task](https://github.com/HumanCellAtlas/skylab/blob/master/library/tasks/HISAT2.wdl) to call HISAT2 and perform graph-based alignment of paired- or single-end reads (in the form of fastq files) to a reference genome. This task  requires a reference index which can be built following the [HCA's build_indices documentation](https://github.com/HumanCellAtlas/skylab/tree/master/library/accessory_workflows/build_indices). The outputs of the task include a genome-aligned BAM file, a BAM index, and an alignment log file. 
+HISAT2 is a fast, cost-efficient alignment tool that can determine the presence of non-transcript sequences and true transcript sequences, taking into account the presence of single-nucleotide polymorphisms ([Kim et al.,2019](https://www.nature.com/articles/s41587-019-0201-4)). The Smart-seq2 Single Sample workflow uses the [HISAT2 task](https://github.com/HumanCellAtlas/skylab/blob/master/library/tasks/HISAT2.wdl) to call HISAT2 and perform graph-based alignment of paired- or single-end reads (in the form of FASTQ files) to a reference genome. This task  requires a reference index which can be built following the [HCA's build_indices](https://github.com/HumanCellAtlas/skylab/tree/master/library/accessory_workflows/build_indices) documentation. The outputs of the task include a genome-aligned BAM file, a BAM index, and an alignment log file. 
```

Comment:
This line also contains two full paths into skylab that could be changed to use relative paths.

Reply:
I changed these two to relative paths.